packages/qdrant-loader/tests/unit/connectors/jira/test_jira_config.py

```
@@ -263,3 +263,219 @@ def test_dc_multiple_placeholders_all_reported(self):
+
+    def test_none_value_accepted(self):
+        """None value should be accepted (fetch all issues)."""
+
+        cfg = _make(updated_after=None)
+        assert cfg.updated_after is None
+
+    def test_datetime_object_accepted(self):
+        """Datetime object should pass through unchanged."""
+        from datetime import datetime
+
+        test_dt = datetime(2026, 5, 4, 12, 0, 0)
```

Ruff (0.15.14)

[warning] `datetime.datetime()` called without a `tzinfo` argument

(DTZ001)